.github/workflows/cicd-main.yml

```diff
@@ -256,11 +256,12 @@ jobs:
           quantization.num_calib_size=8 \
           inference.batch_size=2 \
           export.inference_tensor_parallel=2 \
+          export.sample_output=False \
           export.save_path=/home/TestData/nlp/megatron_llama/ci_fp8.qnemo
       AFTER_SCRIPT: |
         rm -rf /home/TestData/nlp/megatron_llama/ci_fp8.qnemo
 
-  OPTIONAL_L2_PTQ_Llama2_INT8_SQ:
+  L2_PTQ_Llama2_INT8_SQ:
```

Comment:
Please update back to OPTIONAL_L2_PTQ_Llama2_INT8_SQ
It will be optional until confirmed proven that is not flaky anymore

Reply:
👍